packages/authentication/src/__tests__/fixtures/keys.ts

```diff
@@ -0,0 +1,26 @@
+import {BindingKey} from '@loopback/context';
+import {BasicAuthenticationUserService} from './services/basic-auth-user-service';
+import {JWTService} from './services/jwt-service';
+import {UserRepository} from './users/user.repository';
+
+export const USER_REPO = BindingKey.create<UserRepository>(
+  'authentication.user.repo',
+);
+
+export namespace BasicAuthenticationStrategyBindings {
+  export const USER_SERVICE = BindingKey.create<BasicAuthenticationUserService>(
+    'services.authentication.basic.user.service',
+  );
+}
+
+export namespace JWTAuthenticationStrategyBindings {
+  export const TOKEN_SECRET = BindingKey.create<string>(
+    'authentication.jwt.secret',
+  );
+  export const TOKEN_EXPIRES_IN = BindingKey.create<string>(
+    'authentication.jwt.expires.in.seconds',
+  );
+  export const TOKEN_SERVICE = BindingKey.create<JWTService>(
```

Comment:
Because I am not going to create a binding key with generic interface `UserService`, to be consistent, I won't create a binding key using the generic interface `TokenService`.